### PR TITLE
chore(docs): fixed multiple typos

### PR DESCRIPTION
**Pull request by Subash-Lamichhane**
## Checklist

- [x] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [x] Description explains motivation and solution
- [ ] Tests added (always)
- [x] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

Fixed multiple typos of docs folder:

### CODE REVIEW

File: docs/docs/04-standard-library/01-cloud/service.md
Comment (MarkMcCulloh) [inline docs/docs/04-standard-library/01-cloud/service.md:1]: This needs to be updated in the docstrings in https://github.com/winglang/wing/blob/main/libs/wingsdk/src/cloud/service.ts#L181

File: docs/docs/04-standard-library/03-std/api-reference.md
Comment (MarkMcCulloh) [inline docs/docs/04-standard-library/03-std/api-reference.md:1]: Needs to be updated in docstrings in https://github.com/winglang/wing/blob/main/libs/wingsdk/src/std/array.ts#L298

**Review by MarkMcCulloh — CHANGES_REQUESTED** [review]

**Comment by garysassano** [comment]
@eladb What you think about updating stories to correct typos? I remember I did something similar in the past when replacing `Monada` with `Wing` and you asked to revert it.

**Comment by garysassano** [comment]
@eladb What you think about updating stories to correct typos? I remember I did something similar in the past when replacing `Monada` with `Wing` and you asked to revert those.

File: docs/docs/04-standard-library/01-cloud/service.md
Comment (Subash-Lamichhane) [reply docs/docs/04-standard-library/01-cloud/service.md:1]: I have updated it.

**Review by staycoolcall911 — APPROVED** [review]

**Pull request merged**